src/routes/Dashboard/SummaryCard/components/CounterTime.jsx

```diff
@@ -27,6 +27,6 @@ export default function CounterTime({datetime}) {
 }
 
 CounterTime.propTypes = {
-    datetime: PropTypes.string.isRequired
+    datetime: PropTypes.string
```

Comment:
Why did you made this as not required?

It must be required. Put required back.